apps/screenshot-diff/src/computeScreenshotDiff.ts

```diff
@@ -54,19 +59,59 @@ export const computeScreenshotDiff = async (
     }),
   ]);
 
+  await Promise.all(
+    [
+      { ...screenshotDiff.baseScreenshot, filePath: baseScreenshotPath },
+      { ...screenshotDiff.compareScreenshot, filePath: compareScreenshotPath },
+    ]
+      .filter(
+        (screenshot) =>
+          screenshot &&
+          screenshot.fileId &&
+          screenshot.file &&
+          (screenshot.file.width === undefined ||
+            screenshot.file.height === undefined)
+      )
+      .map(async (screenshot) => {
+        const { width, height } = await getImageSize(screenshot.filePath);
```

Comment:
`getImageSize` is already done in the diff compute, so here we will compute the image size twice. We should memoize it to avoid computing size multiple times.